### DISCUSSION TITLE
[REQUEST] Frigate NVR

### DISCUSSION THREAD

**Discussion by VCTGomes** [tteck/Proxmox#1813] [Ideas]
Frigate is an excellent way to integrate cameras into IA, allowing me to do a lot of things, like detecting people, cars and much more.

It'd be cool if there's a simple script to install it, allowing passthrough GPU and Coral CPU for those who have this.

Please, create a script to install Frigate LXC 🙏🏻

**Comment by btehan** [comment]
https://github.com/tteck/Proxmox/discussions?discussions_q=is%3Aopen+frigate
